src/main/ipv-stub/ipv-token.ts

```diff
@@ -98,7 +99,7 @@ function getValidBodyOrThrow(body: string | null): querystring.ParsedUrlQuery {
   }
 
   const authCode = query["code"];
-  if (authCode != AUTH_CODE) {
+  if (!authCode || Array.isArray(authCode) || !authCodeIsValid(authCode)) {
```

Comment:
Why is the Array.isArray() check needed ?

Reply:
It was to appease typescript as `query["code"]` can be of type `String[]` and we don't want that. I've removed it now anyway